simple/stats/constants.py

```diff
@@ -82,3 +82,8 @@
 # DC links
 DC_HOME = "https://datacommons.org"
 DC_BROWSER = "https://datacommons.org/browser"
+
+# "Special" filenames.
+# i.e. if files have these names, they are handled in specific ways.
+VERTICAL_SPECS_FILE_NAME = "dc.vertical_specs.json"
+SPECIAL_FILE_NAMES = set([VERTICAL_SPECS_FILE_NAME])
```

Comment:
Wonder why not refer to a field in the config for the file name instead of assuming a specific filename?

Reply:
Going with the overall theme of keeping the config minimal. Not a strong preference though - I can go either way here. Let me know your preference.

Reply:
Chatted with Bo offline and decided to specify it in config.